api/DHSC.FingertipsNext.Modules.DataManagement.Service/DataManagementService.cs

```diff
@@ -67,7 +74,10 @@ public async Task<UploadHealthDataResponse> UploadFileAsync(Stream fileStream, i
         {
             UploadDebugLog(_logger, batchId, _containerName, null);
             await blobClient.UploadAsync(fileStream);
-            UploadSuccessfulLog(_logger, null);
+            WriteUploadSuccessLog(_logger, originalFileName, _timeProvider.GetUtcNow().UtcDateTime,
+                userId,
+                publishedAt, batchId, indicatorId);
```

Comment:
This linebreak is a bit weird:
```suggestion
                userId, publishedAt, batchId, indicatorId);
```